src/py/flwr/server/strategy/fedavgm.py

```diff
@@ -38,8 +38,40 @@
 from .fedavg import FedAvg
 
 
+# pylint: disable=line-too-long
 class FedAvgM(FedAvg):
-    """Configurable FedAvg with Momentum strategy implementation."""
+    """Federated Averaging with Momentum strategy.
+
+    Implementation based on https://arxiv.org/pdf/1909.06335.pdf
```

Comment:
```suggestion
    Implementation based on https://arxiv.org/abs/1909.06335
```
We could use this opportunity to use the abstract instead of the pdf, although not completely necessary (the url works as is)